src/github/github.py

```diff
@@ -0,0 +1,188 @@
+        org: str = github_org_name,
+        uid: str | None = None,
+        email: str | None = None
+) -> None:
+    if uid is None and email is None:
+        raise Exception("uid and username cannot both be empty")
+    result = None
+    # Arbitrarily prefer uid
+    if uid is not None and email is None:
+        result = await _github_request_post(f"https://api.github.com/orgs/{org}/invitations",
+                               os.environ.get("GITHUB_TOKEN"),
+                               dumps({"invitee_id":uid, "role":"direct_member"}))
+    # Assume at this point both exist, but use email
+    else:
+        result = await _github_request_post(f"https://api.github.com/orgs/{org}/invitations",
+                               os.environ.get("GITHUB_TOKEN"),
+                               dumps({"email":email, "role":"direct_member"}))
+    result_json = result.json()
+    # Logging here potentially?
+    if result.status_code != 201:
+        raise Exception(f"Status code {result.status_code} returned when attempting to add user to org: {result_json['message']}: {[error['message'] for error in result_json['errors']]}")
+
+async def delete_user_from_org(
+        username: str,
+        org: str = github_org_name
+) -> None:
+    if username is None:
+        raise Exception("Username cannot be empty")
+    result = await _github_request_delete(f"https://api.github.com/orgs/{org}/memberships/{username}",
+                                          os.environ.get("GITHUB_TOKEN"))
+    # Logging here potentially?
+    if result.status_code != 204:
+        raise Exception(f"Status code {result.status_code} returned when attempting to delete user {username} from organization {org}")
+
+async def get_teams(
+        org: str = github_org_name
+) -> list[str]:
+    result = await _github_request_get(f"https://api.github.com/orgs/{org}/teams", os.environ.get("GITHUB_TOKEN"))
+    json_result = result.json()
+    return [GithubTeam(team["id"], team["url"], team["name"], team["slug"]) for team in json_result]
+
+async def add_user_to_team(
+        username: str,
+        slug: str,
+        org: str = github_org_name
+) -> None:
+    result = await _github_request_put(f"https://api.github.com/orgs/{org}/teams/{slug}/memberships/{username}",
+                                        os.environ.get("GITHUB_TOKEN"),
+                                        dumps({"role":"member"}))
+    result_json = result.json()
+    # Logging here potentially?
+    if result.status_code != 200:
+        raise Exception(f"Status code {result.status_code} returned when attempting to add user to team: {result_json['message']}")
+
+async def remove_user_from_team(
+        username: str,
+        slug: str,
+        org: str = github_org_name
+) -> None:
+    result = await _github_request_delete(f"https://api.github.com/orgs/{org}/teams/{slug}/memberships/{username}",
+                                          os.environ.get("GITHUB_TOKEN"))
```

Comment:
Not sure how you feel about it, but I think this style of line-break formatting is easier to read
```suggestion
    result = await _github_request_delete(
        f"https://api.github.com/orgs/{org}/teams/{slug}/memberships/{username}",
        os.environ.get("GITHUB_TOKEN"),
    )
```
(I found this in the python google style guide as well)